FAQ.md

```diff
@@ -55,9 +55,11 @@ Note: For storage space reasons, the programming guide is only available in gz c
 Based on [#1797](https://github.com/iptv-org/iptv/issues/1797), [#2438](https://github.com/iptv-org/iptv/issues/2438), [#4537](https://github.com/iptv-org/iptv/discussions/4537) and [#7043](https://github.com/iptv-org/iptv/discussions/7043).
 
 ### Is it possible to view channels directly from the main website?
-No, you will only see information about active and inactive transmissions of channels.
+No. In own website you will only see an extensive list of channels and their respective information, like a global database. However, you will be able to search and review the channels you want, and if your favorite channel is legally available as an indexed stream in the IPTV playlist.
```

Comment:
```suggestion
No. In our website, you will only see an extensive list of channels and their respective information, like a global database. In this case, you will only be able to search and review the channels you want. If your favorite channel is legally available as an indexed stream in the IPTV playlist, there's a "Streams" option that will be available on the right side of the channel, so that you can have the link to watch it. Else, you can make an issue to ask for it. Please respect the [contribution guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md) before doing so;
```

Reply:
Thank you for improving the explanation.